tests/data_utils_tests/data_tests.py

```diff
@@ -84,6 +85,30 @@ def test_multiply_transform(self):
         self.assertEqual(len(result["A"]), len(self.df) * n_repeats)
 
 
+class TestTokenCounterTransform(unittest.TestCase):
+    def setUp(self):
```

Comment:
are there any failure cases for the function? maybe non-str data types?

Reply:
discussed offline.